eclass/desktop.eclass

```diff
@@ -16,29 +16,29 @@ case ${EAPI} in
 esac
 
 # @FUNCTION: make_desktop_entry
-# @USAGE: <command> [name] [icon] [categories] [fields]
+# @USAGE: <command> [name] [icon] [categories] [entries]
 # @DESCRIPTION:
 # Make a .desktop file.
 #
 # @CODE
-# binary:     what command does the app run with ?
-# name:       the name that will show up in the menu
-# icon:       the icon to use in the menu entry
+# command:    Exec command the app is being run with, also base for TryExec
+# name:       Name that will show up in the menu; defaults to PN
+# icon:       Icon to use with the menu entry; defaults to PN
 #             this can be relative (to /usr/share/pixmaps) or
 #             a full path to an icon
 # categories: Categories for this kind of application. Examples:
 #             https://specifications.freedesktop.org/menu-spec/latest/apa.html
 #             if unset, function tries to guess from package's category
-# fields:     extra fields to append to the desktop file; a printf string
+# entries:    Key=Value entry to append to the desktop file; a printf string
 # @CODE
 make_desktop_entry() {
 	[[ -z $1 ]] && die "make_desktop_entry: You must specify the executable"
 
-	local exec=${1}
+	local cmd=${1}
```

Comment:
this rename could be its own patch, to keep things atomic and have "exec" in the commit title to find it later if need be

But i do not feel too hard about that split